configuration/configuring-root-certificates/en.md

```diff
@@ -0,0 +1,119 @@
+```
+
+### Validate the Certificate
+
+`$ certutil -d sql:$HOME/.pki/nssdb -V -u <certusage> -n <cert_name>`
+
+#### Example Output
+
+In this example, we specify the usage context with the `-u` option to "Verify CA" `Y`.
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -V -n mynew-Root-CA -u Y
+certutil: certificate is valid
+```
+
+In this example, we try to validate the Root/CA certificate in the SSL client usage context.
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -V -n mynew-Root-CA -u C
+certutil: certificate is invalid: Certificate key usage inadequate for attempted operation.
+```
+
+### Using a Certificate
+
+Some client applications will need to be restarted to use the new CA certificate. Valid SSL or TLS host/service certificates including a signature with by the newly installed CA/Root certificate can now be validated and inspected. Web browsers provide utilities for certificate inspection, often with a status icon in the URL bar. 
+
+A command line method of inspecting a certificate: `curl https://protected.resource.uri -vI --stderr -`
+
+### Deleting a Certificate in the Database
+
+`$ certutil -d sql:$HOME/.pki/nssdb -D -n <cert-name>`
+
+#### Example Output
+
+```
+$ certuilt -d sql:$HOME/.pki/nssdb -L
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+```
+
+## Advanced Topics
+
+### Certificate Types
+
+SSL/TLS and SSH certificates are separate and unique concepts with distinct use-cases. This article did not provide details on hosting or serving network resources with a non-public certificate, as most implementations are service dependent (eg: Apache vs MongoDB) and are generally independent from the Solus' Public/Private Key Infrastructure (PKI).
+
+Read the certutil help documentation for more details on the available types of certificates that can be managed this way.
+
+### Self-signed Certificate Alternatives
+
+If your goal is to simply conform with modern browser security standards for local resources, for example in a home lab, free certificate services are publicly available that can integrate with many services that will greatly simplify the certificate management process. No matter what certificate management  method is used, there are trade offs that must be considered and great caution is recommended before presenting resources to unsafe networks like the internet. Issues like expired certificates have impacted major internet services, and can prevent access to critical services or resources. 
+
+### Installing CA Certificates in the Browser
+
+It may be possible to install a CA cert into a web browser directly. This will not allow other clients access to the certificate and many types of certificates can not be installed this way and will result in a confusing message like "This certificate is already installed as a certificate authority", although no certificate has been installed.
```

Comment:
```suggestion
It may be possible to install a CA certificate into a web browser directly. This will not allow other clients access to the certificate and many types of certificates can not be installed this way and will result in a confusing message like "This certificate is already installed as a certificate authority", although no certificate has been installed.
```

Since the rest of the article always uses the full word "certificate", I think it's best to do it here as well.